### PR TITLE
Fixing Deadlinks

### PR DESCRIPTION
**Pull request by ShambhaviCodes**

### CODE REVIEW

File: guides.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This section will be dedicated to guides to learning Data Science, Machine Learn
   - [Contribute to Virgili0](#contribute-to-virgili0)
   - [👉 Feynman Learning Technique 👈](https://www.linkedin.com/posts/asif-bhat_neverstoplearning-asifbhat-activity-6639250916010717185-avMT)
   - [👉 How to Learn #AI 👈](https://www.linkedin.com/posts/asif-bhat_datasciencepost-on-instagram-artificialintelligence-activity-6634407139324588032-DD4K)
-  - [👉 Python to Neural Networks : A Guide for Beginners 👈] (https://www.linkedin.com/feed/update/urn:li:activity:6690911303441219584/)
+  - [👉 Python to Neural Networks : A Guide for Beginners 👈](https://www.linkedin.com/feed/update/urn:li:activity:6690911303441219584/)
```
Comment (neomatrix369) [inline guides.md:24]: 👍

File: guides.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ This section will be dedicated to guides to learning Data Science, Machine Learn
 ### Existing guides
 
 #### Learning to learn
-  - [Virgilio's Teaching Strategy - Learning to Learn](https://github.com/virgili0/Virgilio/blob/master/Topics/teaching.md)
+  - [Deadlink] [Virgilio's Teaching Strategy - Learning to Learn](https://github.com/virgili0/Virgilio/blob/master/Topics/teaching.md)
```
Comment (neomatrix369) [inline guides.md:35]: Hey @ShambhaviCodes great you pointed out its a deadlink but would even more fabulous if we could find working links and replace these non-working ones.

We have access to both the github repo and their website for this - I'm happy to help here as it's my project as well

**Comment by neomatrix369** [comment]
Associated with issue #79

File: guides.md (hunk 1 above)
Comment (neomatrix369) [reply guides.md:24]: Is it an end-to-end guide? if so you could mention in the description.

**Comment by neomatrix369** [comment]
@ShambhaviCodes any chance looking at the comments on this PR?

**Comment by ShambhaviCodes** [comment]
@neomatrix369 Was caught up with multiple things, will look into this hopefully by the end of this week!

**Comment by neomatrix369** [comment]
Let's close this PR as discussed, happy if you open a new one working on another topic.

**Pull request closed without merging**